### PR TITLE
Add Fesapi link

### PR DESCRIPTION
**Pull request by philippeVerney**
Since Resqml can deal with data from seismic to simulation, I created the category "Data Management".

### CODE REVIEW

**Comment by EvanBianco** [comment]
@phillipeVerney, thanks for the contribution here! This repo looks interesting. Looking forward to checking it out some more.

Do you think we could park this under the Reservoir Engineering heading?

I realize there is likely more in this repo than "just ResEng stuff", and might have components of production engineering, and drilling engineering stuff (WITSML and so on). However the reason I suggest we put it under Reservoir Engineering is that many of the other libraries have I/O aspects to them: segyio, welly, lasio, obspy, etc. but we haven't parked them in a pure Data Management category.

Thoughts?

Indeed it is hard to categorize this stuff definitively as several libraries fall into more than one of the categories we've defined here. Fun to think about and make up as we all go along!

**Comment by philippeVerney** [comment]
Thanks for your quick comment.

Yes, if we don't open a new category and if we restrict item to be in a single category, I think "Reservoir Engineering" heading is the best option. Indeed, "RES" part of "RESQML" stands for RESERVOIR and most of RESQML adopters have the reservoir in mind.
However, RESQML really wants to carry all information which are usually upstream to the reservoir domain in order not to loose information at reservoir engineering stage. That's one of the goal of RESQML.
"Data management" is really a transverse category which I think becomes more and more present in companies. Think about Petrel studio, coming Roxar/Emerson OpenDB, Openspririt, etc.. all these initiative cannot fall in a single geoscience domain category : they are really transverse.

To summarize:
- Yes it can park in Reservoir Engineering heading. It is a bit restrictive but it is not at all a big problem.
- Longer term, I think that having a "Data management" category would be nice

**Comment by EvanBianco** [comment]
I agree a Data Management category would be nice in the future. Maybe we can make a Data Management tag or badge and add it to the relevant libraries. Please go ahead and move this into the Res Eng section of the Readme. Cheers

**Comment by JesperDramsch** [comment]
Could also go into platform.
